### PR TITLE
Remove CSS and restructure

### PR DESCRIPTION
**Pull request by stefvhuynh**

### CODE REVIEW

**Comment by boygirl** [comment]
@stefvhuynh I created a branch called rewrite to merge this into instead of master that we can use as the long running branch for completing the rewrite.

I'm going to tag @BrittanyIRL and @treyhoover on this PR to help with the handoff.

File: src/styles/prism.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
+import { css } from "styled-components";
+
```
Comment (boygirl) [inline src/styles/prism.js]: I think you may be able to include this from the prism js package instead

File: static-config-parts/static-webpack-config.js [hunk 2]
```diff
@@ -1,105 +0,0 @@
...
-                ],
-                flexbox: "no-2009"
-              })
-            ]
-          }
-        }
-      ]
-    },
-    {
-      test: /\.(svg)(\?.*)?$/,
```
Comment (boygirl) [inline static-config-parts/static-webpack-config.js]: I think the svg loader may still be required. When running `yarn build` and `yarn serve` on this branch, svgs are not rendering:
Screen Shot 2020-01-05 at 3 36 49 PM

File: src/partials/sidebar/index.js [hunk 3]
```diff
@@ -1,11 +1,15 @@
-import React from "react";
+import React, { Fragment } from "react";
```
[the remark this answers is not in the document]
Comment (stefvhuynh) [reply src/partials/sidebar/index.js]: looks like we aren't advanced enough to use that syntax :/ would need to update some deps

File: src/partials/header.js [hunk 4]
```diff
@@ -1,61 +1,178 @@
...
+
+  > svg {
+    height: 24px;
+    width: 18px;
+  }
+`;
+
+const FormidableLogo = styled(SVG)`
+  display: none;
+  height: 28px;
```
[the remark this answers is not in the document]
Comment (stefvhuynh) [reply src/partials/header.js]: great question! the app was already using a mix of rems and px. we should decide if that is acceptable or if we should move everything to rems. opinions, @boygirl?

[the remark this answers is not in the document]
Comment (boygirl) [reply src/partials/header.js]: I don't have a strong opinion here.

[the remark this answers is not in the document]
Comment (stefvhuynh) [reply src/partials/header.js]: okay, something i've done on past projects is to do
```
html {
  font-size: 62.5%;
}
```
and then use `rem` for all sizing. by setting that percentage, `1px` = `0.1rem` if the user's base font is `16px` (which is the usual setting). so in the design, if we see a certain `px` setting, we just have to divide that number by 10 to get the `rem` value.

i can add this change in this branch.

[the remark this answers is not in the document]
Comment (BrittanyIRL) [reply src/partials/header.js]: i'd love that change. i think it'll bode well moving forward too.

File: src/styles/prism.js (hunk 1 above)
Comment (stefvhuynh) [reply src/styles/prism.js]: but would we be able to use the custom colors from `theme` if we did that? do they have a way to do the theming?

and if we just use this file, do we need the prism dep?

File: src/partials/header.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (stefvhuynh) [reply src/partials/header.js]: ok, i've changed all the files that i touched from `px` to `rem` using the above technique. i did _not_ do this in the files that i didn't work on. so when ya'll go through the style redesign, you'll just have to remember to replace them with `rem`s. if anything, you'll probably just throw out the old styles and start from scratch anyway.

**Comment by boygirl** [comment]
@stefvhuynh travis was not running after I cahnged the base branch on this PR to `rewrite` I pushed a couple commits that 1) add `rewrite` to the list of branches travis builds on and 2) fix some lingering lint issues that were failing the build. I'll merge after CI completes.

**Comment by stefvhuynh** [comment]
> @stefvhuynh travis was not running after I cahnged the base branch on this PR to `rewrite` I pushed a couple commits that 1) add `rewrite` to the list of branches travis builds on and 2) fix some lingering lint issues that were failing the build. I'll merge after CI completes.

Thanks so much!

**Pull request merged**